### ISSUE TITLE
For input string: "12.5"

### ISSUE THREAD

**Issue by sumin5784** [igvteam/igv#1343]
Hello,

I'm trying to install IGV but keep having issues.
If I open IGV, then it gives me an error loading genome data, like hg19 or whatever.
So I load genome sequence file my self and then open gene annotation file (BED), RNAseq bigwig files, and BAM files. 
But it keep giving me an error like:
`
Error loading ~FileName: For input string: "12.5"
`

Whatever file I tried loading, this error **For input string: "12.5"** keep popped out.
I tried install multiple versions in addition to latest version but issue is the same. 
Any help would be appreciated, thanks.

**Comment by jrobinso** [comment]
I need some exact steps to reproduce, including a test file.

**Comment by jrobinso** [comment]
It looks like you have a corrupt "prefs.properties" file in your IGV folder.   Did you edit this file?  You say "I am trying to install IGV",  where did you install it from?

The simplest fix is to shutdown IGV, delete "prefs.properties", and startup again.   If you want to keep your preferences you can edit the file in a plain text editor and either delete the line starting with "DEFAULT_FONT_SIZE", or set the value to an integer.

**Comment by jrobinso** [comment]
Update -- it looks like IGV allowed storing non-integer values of the font size field, which Java does not allow.   This has been addressed.  Regardless the fix is the same.    Its still a mystery how you got this value from a fresh install, perhaps you had used IGV in the past.   For a completely clean break delete the entire "igv" folder while IGV is shutdown.

**Comment by sumin5784** [comment]
Hi Jim,

It worked like charm, thank you so much. 
I used IGV uninstall program and re-installed, error was the same. But the "prefs.properties" file was in document/igv, and igv uninstall program removes only Programs and files/IGV folder. The remained "prefs.properties" in document/igv folder has kept causing the error so far. I completed removed this folder too in addition to uninstall program, and it is working well now. Thanks again!
